c/include/libsbp/sbp_msg_type.h

```diff
@@ -47,6 +47,7 @@ extern "C" {
 #endif
 
 typedef enum {
+  SbpMsgInvalid = -1,
```

Comment:
I define the invalid enum to be "-1", following this pattern elsewhere in our codebase:

https://github.com/swift-nav/libswiftnav/blob/c580b9d041a7b6727ea7efdfa3fc9e6443e902f0/include/swiftnav/signal.h#L177

Reply:
All of these files are autogenerated and will be overwritten then next time the schema is modified and needs to be regenerated. To avoid this you'll need to add these to the templates in `generator/sbpg/target/resources/c/` and run `make c` in the repo root